plugin/src/main/java/org/eobjects/build/AbstractDotnetTestMojo.java

```diff
@@ -21,6 +21,12 @@
     public void executeInternal() throws MojoFailureException {
         final PluginHelper helper = getPluginHelper();
         ArrayList<String> argsList = new ArrayList<String>(Arrays.asList("dotnet", testCommand, "-c", helper.getBuildConfiguration()));
+        
+        if (helper.getBuildTargetFramework() != "")
+        {
```

Comment:
Should probably do a null check and `.equals("")` instead. And the curly braces go on the line of the `if` :)

Reply:
Actually, the null-safe way of doing this comparison would be `if (!"".equals(helper.getBuildTargetFramework()) {`